src/crypto/hsm/CHIPCryptoPALHsm.h

```diff
@@ -146,6 +147,10 @@ class P256KeypairHSM : public P256Keypair
 
     uint32_t GetKeyId(void) { return keyid; }
 
+    CHIP_ERROR CreateOperationalKey(FabricIndex fabricIdx);
+
+    void SetOperationalKeyId(FabricIndex fabricIdx);
```

Comment:
This interface doesn't feel sensible except for the case where the HSM only supports one key and we are setting the fabric-index mapping for that one key.

It also feels inflexible / risky that this does not return CHIP_ERROR.

Reply:
Hi. Removed the interface as of now, as it was not used yet. Will add this in other PR considering other use cases.